### DISCUSSION TITLE
how Lists works with compact-csv ?

### DISCUSSION THREAD

**Discussion by callmephil** [slang-i18n/slang#127] [Q&A] [answered]
I'd like to use the [list feature](https://pub.dev/packages/slang#-lists) provided by slang,
However, I'm using CSV instead of JSON and I can't find the documentation for that.

Has anyone gone through this before?

**Comment by Tienisto** [comment, marked as answer]
In [File Types/CSV](https://github.com/Tienisto/slang#-file-types), you can see that the keys have a number. The number represents the index of the list.

```csv
pages.0,First Page
pages.1,Second Page
```

In the example above, `pages` is a list containing 2 items `0` and `1`.
Just add a third column for your second locale.
